drivers/serial/Kconfig.mcux_lpsci

```diff
@@ -5,19 +5,16 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-menuconfig UART_MCUX_LPSCI
+config UART_MCUX_LPSCI
 	bool "MCUX LPSCI driver"
 	depends on HAS_MCUX_LPSCI && CLOCK_CONTROL
 	select SERIAL_HAS_DRIVER
 	select SERIAL_SUPPORT_INTERRUPT
 	help
 	  Enable the MCUX LPSCI driver.
 
-if UART_MCUX_LPSCI
-
-menuconfig UART_MCUX_LPSCI_0
+config UART_MCUX_LPSCI_0
 	bool "UART 0"
+	depends on UART_MCUX_LPSCI
```

Comment:
`UART_MCUX_LPSCI` should probably stay `menuconfig` as `UART_MCUX_LPSCI_0` is its dependant.

Reply:
Oops... was deliberate, but forgot to mention it.

Was thinking it might be better to show `UART_MCUX_LPSCI_0` indented (which is what you get without `menuconfig`), since the menu would just have that symbol in it otherwise.

Reply:
Changed `UART_MCUX_LPSCI` back to a `menuconfig`.

More important to remove the pointless ones. Rest is pretty subjective.